ibm-build-lab/VAD-VAR-Workshop
# feat: move to nextjs

Relates to https://github.com/ibm-build-lab/vest-content/issues/140 https://github.com/ibm-build-lab/vest-content/issues/139 https://github.com/ibm-build-lab/vest-content/issues/138

Content changes:

- Changed some of the watsonx READMEs to showcase new components in action

Code changes:

- Site overhaul
   -  Now using Next.js

Testing:

1. Update node version to at least 18.18.0
2. install deps with `npm ci`
3. Start locally with `npm run dev`
4. Verify site at http://localhost:3000
5. Test prod build with `npm run build:local` (remote assets won't work with normal build since paths changed)
6. Serve prod locally with `npm run serve`
7. Verify at http://localhost:3000
8. Ensure parity with current site functionality
